result/egressonlytracking.yaml

```diff
@@ -0,0 +1,290 @@
+components:
+  schemas:
+    EgressOnlyTracking.Metrics.Request:
+      description: >-
+        The container for a egress only tracking metric request.
+      type: object
+      properties:
+        port_names:
+          description: |-
+            Egress only tracking metrics will be retrieved for these port names.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/EgressOnlyTracking/properties/port_name"
+          x-field-uid: 1
+        tagged_metrics:
+          $ref: '#/components/schemas/EgressOnlyTracking.TaggedMetrics.Filter'
```

Comment:
Cant we put the fields directly here ? What is the advantage of a separate class ?